Client/Frontend/Home/Wallpapers/Legacy/LegacyWallpaperDataManager.swift

```diff
@@ -28,11 +28,12 @@ private struct LegacyWallpaperCollection {
     /// region or time limited.
     ///
     /// - Parameters:
-    ///   - names: An array of the names of the wallpapers included in the collection.
+    ///   - wallpaperFileNames: An array of the names of the wallpapers included in the collection.
     ///   - type: The collection type.
-    ///   - expiryDate: An optional expiry date, on and after which the wallpapers in
     ///         the array are no longer shown. If May 1, 2022, the collection is no
     ///         longer visible on May 1, 2022.
```

Comment:
Those lines were part of the `expiryDate` explanation. Should be removed I think
```suggestion
```